### ISSUE TITLE
Build a small POC with shoelace for UI elements inside of WP

### ISSUE THREAD

**Issue by arzola** [pressbooks/ideas#537]
We want to standarize / harmonize our UI elements. We've been testing Shoelace and think it might be a good fit for our products, WP custom admin pages, book pages and other standalone products like the Pressbooks Directory

@SteelWagstaff shared https://css-tricks.com/using-web-components-in-wordpress-is-easier-than-you-think/

DoD: 
- [ ] Work on a POC with Shoelace and see how it would work in our WP solutions
- [ ] Document findings, pros and cons and share with the team
- [ ] Create next steps issues

**Comment by cmurtagh** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pressbooks-Product-Backlog-60ae7d807f6e370010b3d5be/issues/pressbooks/pressbooks/3784?planning-poker) with Zenhub @arzola @fdalcin @ho-man-chan @MichelleWeremczuk @richard015ar @tw77

**Comment by richard015ar** [comment]
It is not clear to me the DoD for this task, so I can size it.
Do we want a document as a result of research? a proposal/agreement?

**Comment by cmurtagh** [comment]
Yeah, I think that makes sense. Let's make it a small research and scoping project.
